<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageChooser</class>
 <widget class="QWidget" name="ImageChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ImageLineEdit" name="lineEditFile">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>38</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/images/res/images/light/appbar.folder.open.png</normaloff>:/images/res/images/light/appbar.folder.open.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="infoBtn">
     <property name="toolTip">
      <string>Dump headers from an application image</string>
     </property>
     <property name="statusTip">
      <string>Dump headers from an application image</string>
     </property>
     <property name="whatsThis">
      <string>Dump headers from an application image</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/images/res/images/light/appbar.information.png</normaloff>:/images/res/images/light/appbar.information.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="HexLineEdit" name="lineEditOffset">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HexLineEdit</class>
   <extends>QLineEdit</extends>
   <header>hexlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ImageLineEdit</class>
   <extends>QLineEdit</extends>
   <header>imagelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
